<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MngFormUom</class>
 <widget class="QDialog" name="MngFormUom">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Manutenção em Unidades de Medidas</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../images.qrc">
    <normaloff>:/images/PNG/128x128/gnome-system-config-128x128.png</normaloff>:/images/PNG/128x128/gnome-system-config-128x128.png</iconset>
  </property>
  <widget class="QToolButton" name="TButtonExitUom">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>90</width>
     <height>80</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Sair da Janela de Manutenção...</string>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/images/PNG/48x48/Exithome-48x48.png</normaloff>:/images/PNG/48x48/Exithome-48x48.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>48</width>
     <height>48</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
  </widget>
  <widget class="QGroupBox" name="GBoxViewUom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>731</width>
     <height>341</height>
    </rect>
   </property>
   <property name="title">
    <string>Unidades de Medidas</string>
   </property>
   <widget class="QTableView" name="TableViewUom">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>591</width>
      <height>291</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Unidades de Medidas Disponíveis...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonRemoveUom">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Fazer a Remoção do Item Selecionado da Tabela...</string>
    </property>
    <property name="text">
     <string>Remover</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/48x48/delete-48x48.png</normaloff>:/images/PNG/48x48/delete-48x48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonCleanUom">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>80</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Limpar os Campos de Cadastro e Atualizar a Janela de Manutenção...</string>
    </property>
    <property name="text">
     <string>Limpar</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/48x48/Rebuild-48x48.png</normaloff>:/images/PNG/48x48/Rebuild-48x48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="GBoxDataUom">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>731</width>
     <height>131</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="title">
    <string>Dados</string>
   </property>
   <widget class="QLabel" name="LabelNameUom">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Descrição:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="LabelUnitUom">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Unidade:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QToolButton" name="TButtonSaveUom">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>90</width>
      <height>80</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Salvar os dados para a Unidade de Medida...</string>
    </property>
    <property name="text">
     <string>Salvar</string>
    </property>
    <property name="icon">
     <iconset resource="../images.qrc">
      <normaloff>:/images/PNG/48x48/Filesave-48x48.png</normaloff>:/images/PNG/48x48/Filesave-48x48.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>48</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextUnderIcon</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditIDUnitUom">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>30</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditNameUom">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>80</y>
      <width>230</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar uma  Descrição/Nome para a Unidade de Medida...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LEditUnitUom">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Informar a Unidade de Medida para o Produto...</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="GBoxStatusActiveUom">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>171</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>Status</string>
    </property>
    <widget class="QCheckBox" name="CheckBoxStatusActiveUom">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Ativar/Desativar a Unidade de Medida para ser ou não Usada...</string>
     </property>
     <property name="text">
      <string>Ativa</string>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="CheckBoxIsUnitOfWeight">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Marcar!, caso Seja uma Unidade de Medida para Peso...</string>
    </property>
    <property name="text">
     <string>É uma Unidade de Peso?</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections/>
</ui>
